packages/client/src/schema-component/antd/slider/slider.tsx

```
@@ -0,0 +1,21 @@
+import { connect, mapProps, mapReadPretty } from '@tachybase/schema';
+
+import { Slider as AntSlider } from 'antd';
+
+import { ReadPretty } from '../input-number/ReadPretty';
+
+type SliderProps = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)